src/foam/comics/v2/DAOUpdateView.js

```diff
@@ -59,6 +59,13 @@ foam.CLASS({
       class: 'FObjectProperty',
       name: 'data'
     },
+    {
+      class: 'FObjectProperty',
+      name: 'workingData',
+      expression: function(data) {
+        return data.clone(this.__subContext__)
```

Comment:
Ditch the __subContext__

Reply:
Done just left it as
`data.clone(this)`